src/midi/midi_mt32.cpp

```diff
@@ -483,111 +611,89 @@ static std::set<const LASynthModel*> populate_available_models(
 // across the first row and directories are printed down the left column.
 // Long directories are truncated and model versions are used to avoid text
 // wrapping.
-MIDI_RC MidiHandler_mt32::ListAll(Program *caller)
+MIDI_RC MidiHandler_mt32::ListAll(Program* caller)
 {
 	// Table layout constants
-	constexpr char column_delim[] = " ";
-	constexpr char indent[] = "  ";
-	constexpr char trailing_dots[] = "..";
-	const auto delim_width = strlen(column_delim);
-
-	const LASynthModel* models_without_aliases[] = {&cm32ln_100_model,
-	                                                &cm32l_102_model,
-	                                                &cm32l_100_model,
-	                                                &mt32_107_model,
-	                                                &mt32_106_model,
-	                                                &mt32_105_model,
-	                                                &mt32_104_model,
-	                                                &mt32_bluer_model,
-	                                                &mt32_204_model,
-	                                                &mt32_207_model,
-	                                                &mt32_206_model,
-	                                                &mt32_203_model};
-
-	const size_t max_dir_width = get_max_dir_width(models_without_aliases,
-	                                               indent, column_delim);
-	const size_t truncated_dir_width = max_dir_width - strlen(trailing_dots);
-	assert(truncated_dir_width < max_dir_width);
+	constexpr char column_delim[]  = " ";
+	constexpr char indent[]        = "  ";
+
+	const std::vector<const LASynthModel*> mt32_models = {&mt32_104_model,
+	                                                      &mt32_105_model,
+	                                                      &mt32_106_model,
+	                                                      &mt32_107_model,
+	                                                      &mt32_bluer_model,
+	                                                      &mt32_203_model,
+	                                                      &mt32_204_model,
+	                                                      &mt32_206_model,
+	                                                      &mt32_207_model};
+
+	const std::vector<const LASynthModel*> cm32_models = {&cm32l_100_model,
+	                                                      &cm32l_102_model,
+	                                                      &cm32ln_100_model};
+
```

Comment:
These local `mt32_models` and `cm32_models` vectors have a name conflict with the file-scope variables.

Reply:
They just shadow them 😄 , but probably best to avoid that when we're in full control over namings.